DxDispatch/src/model/JsonParsers.h

```diff
@@ -88,6 +88,9 @@ namespace JsonParsers
     DML_EXECUTION_FLAGS ParseDmlExecutionFlags(const rapidjson::Value& value);
     DML_EXECUTION_FLAGS ParseDmlExecutionFlagsField(const rapidjson::Value& object, std::string_view fieldName, bool required = true, DML_EXECUTION_FLAGS defaultValue = DML_EXECUTION_FLAG_NONE);
 
+    // DmlCompileType
+    DmlCompileType ParseDmlCompileTypeField(const rapidjson::Value& object, std::string_view fieldName, bool required = true, DmlCompileType defaultValue = DmlCompileType::DmlCompileOp);
```

Comment:
nit: this isn't a directml public API enum (see comment above). I'd just remove the declaration from JsonParsers.h since it's not something external code is likely to test or reuse outside the scope of parsing the whole dispatchable description.